src/mono/wasm/Wasm.Build.Tests/BrowserRunner.cs

```diff
@@ -91,7 +91,8 @@ string args
     ) {
         var url = new Uri(browserUrl);
         Playwright = await Microsoft.Playwright.Playwright.CreateAsync();
-        string[] chromeArgs = new[] { $"--explicitly-allowed-ports={url.Port}" };
+        // ignore certificate error -> Microsoft.Playwright.PlaywrightException : net::ERR_CERT_AUTHORITY_INVALID
```

Comment:
Coud you use HTTP instead?

Reply:
This error, if being thrown on http might be caused by other factors like firewall/proxy, network connectivity etc. For me it happens on codespaces only, on local Windows machine it's fine. I would prefer to avoid it failing in codespaces with runtime code not-connected error.